dart_io_comparison.md

```diff
@@ -2,6 +2,18 @@
 
```

Comment:
Add a sentence that this page covers the API differences and performance differences.